package.json

```diff
@@ -56,6 +56,7 @@
     "jstat": "^1.5.2",
     "lodash": "^4.11.1",
     "numeric": "^1.2.6",
-    "pify": "^2.3.0"
+    "pify": "^2.3.0",
+    "distributions":"^1.0.0"
```

Comment:
Going to re-add the _distributions_ package via the CLI. This will modify _package.json_.